about.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: About
+permalink: /about/
+---
+
+This reference is currently a work in progress. Not all pages may be complete and pages may be updated at any time without notice.
```

Comment:
A "this page was last updated on X, see the complete change log at Y" would be pretty sweet.